addons/account_payment/tests/test_payment_refund_wizard.py

```diff
@@ -0,0 +1,35 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.account_payment.tests.common import AccountPaymentCommon
+from odoo.tests import tagged
+
+
+@tagged('-at_install', 'post_install')
+class TestPaymentRefundWizard(AccountPaymentCommon):
+
+    def setUp(self):
+        super().setUp()
+        self.admin_user = self.env.ref('base.user_admin')
+        self.demo_user = self.env['res.users'].search([('login', '=', 'demo')])
```

Comment:
```suggestion
        self.demo_user = self.internal_user
```

Tests shouldn't rely on demo data, to run, and `demo` user doesn't exist without demo data. Instead you can use `internal_user` which was defined in one of the inherited test classes.